jenkins/internal-ci/generic/pr/cortx_deployment/cortx-rgw.groovy

```diff
@@ -171,7 +172,7 @@ pipeline {
                 script { build_stage = env.STAGE_NAME }
                 script {
                     try {
-                        def buildCortxAllImage = build job: 'cortx-docker-images-for-PR', wait: true,
+                        def buildCortxAllImage = build job: '/Cortx-PR-Build/Cortx-Deployment/Generic/cortx-docker-images-for-PR', wait: true,
```

Comment:
Is there any specific need to change job details

Reply:
Done